<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateQuestionWindow</class>
 <widget class="QWidget" name="CreateQuestionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание теста</string>
  </property>
  <widget class="QPushButton" name="proverka_batton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TeacherWindow</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="oneTrueAnswerButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>один правильный ответ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="severalTruAnswerButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Несколько правильных ответов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAnswerButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>создать ответ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveQuestionButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>сохранить вопрос</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>281</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>180</y>
     <width>281</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QPushButton" name="deleteAnswerButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить ответ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberAnswerEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityAnswerEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveTestButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>сохранить тест</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityQuestionEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
